src/Last/README.md

```diff
@@ -0,0 +1,453 @@
+or a function that returns an instance of `First`. When passed the instance,
+a transformed `Last` is returned. When passed a `First` returning function,
+a function will be returned that takes a given value and returns a `Last`.
+
+```javascript
+const Last  = require('crocks/Last')
+const First = require('crocks/First')
+const firstToLast = require('crocks/Last/firstToLast')
+
+const isString = require('crocks/predicates/isString')
+const mconcatMap = require('crocks/helpers/mconcatMap')
+const safe = require('crocks/Maybe/safe')
+
+// lastString :: [ a ] -> Last String
+const firstString =
+  mconcatMap(First, safe(isString))
+
+// unfixFirstString :: [ a ] -> First String
+const unfixFirstString =
+  firstToLast(firstString)
+
+firstToLast(First.empty())
+//=> Last( Nothing )
+
+firstToLast(First(false))
+//=> Last( Just false )
+
+unfixFirstString([ 'one', 2, 'Three', 4 ])
+  .concat(Last('another string'))
+//=> Last( Just "another string" )
+
+unfixFirstString([ 1, 2, 3, 4 ])
+  .concat(Last('Last String'))
+//=> Last( Just "Last String" )
+```
+
+#### maybeToLast
+
+`crocks/Last/maybeToLast`
+
+```haskell
+maybeToLast :: Maybe a -> Last a
+maybeToLast :: (a -> Maybe b) -> a -> Last b
+```
+
+Used to transform a given `Maybe` instance to a `Last` instance, `maybeToLast`
+will turn a `Just` into a non-empty `Last` instance, wrapping the original value
+contained within the `Last`. All `Nothing` instances will map to
+an [`empty`](#empty) `Last` instance.
+
+This function is available mostly for completion sake, as `Last` can always
+take a `Maybe` as its argument during construction. So while there is not a
+real need for this to be used for transforming instances, it can come in
+handy for lifting `Maybe` returning functions.
+
+Like all `crocks` transformation functions, `maybeToLast` has (2) possible
+signatures and will behave differently when passed either a `Maybe` instance
+or a function that returns an instance of `Maybe`. When passed the instance,
+a transformed `Last` is returned. When passed a `Maybe` returning function,
+a function will be returned that takes a given value and returns a `Last`.
+
+```javascript
+const Last = require('crocks/Last')
+const { Nothing, Just } = require('crocks/Maybe')
+const maybeToLast = require('crocks/Last/maybeToLast')
+
+const chain = require('crocks/pointfree/chain')
+const compose = require('crocks/helpers/compose')
+const isNumber = require('crocks/predicates/isNumber')
+const prop = require('crocks/Maybe/prop')
+const safe = require('crocks/Maybe/safe')
+
+// numVal :: a -> Maybe Number
+const numVal = compose(
+  chain(safe(isNumber)),
+  prop('val')
+)
+
+// lastNumVal :: a -> Last Number
+const lastNumVal =
+  maybeToLast(numVal)
+
+maybeToLast(Just(99))
+//=> Last( Just 99 )
+
+maybeToLast(Nothing())
+//=> Last( Nothing )
+
+Last(Just(99))
+//=> Last( Just 99 )
+
+Last(Nothing())
+//=> Last( Nothing )
+
+lastNumVal({ val: 97 })
+  .concat(Last(80))
+//=> Last( Just 97 )
```

Comment:
Not sure what this example is supposed to be, but the result is incorrect. The following example is exactly the same but has the correct result.

Reply:
That should have been switched to:
```js
Last(Just(80))
  .concat(lastNumVal({ val: 97 }))
//=> Last( Just 97 )
```